From: Andrew Morton <akpm@linux-foundation.org>

fs/cifs/cifssmb.c: In function 'CIFSSMBNegotiate':
fs/cifs/cifssmb.c:444: error: 'CIFSSEC_MAY_LANMAN' undeclared (first use in this function)
fs/cifs/cifssmb.c:444: error: (Each undeclared identifier is reported only once
fs/cifs/cifssmb.c:444: error: for each function it appears in.)
fs/cifs/cifssmb.c:444: error: 'CIFSSEC_MAY_PLNTXT' undeclared (first use in this function)

With http://userweb.kernel.org/~akpm/config-sony.txt

Cc: Steven French <sfrench@us.ibm.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 fs/cifs/cifsglob.h |    3 +++
 1 file changed, 3 insertions(+)

diff -puN fs/cifs/cifsglob.h~cifs-build-fix fs/cifs/cifsglob.h
--- a/fs/cifs/cifsglob.h~cifs-build-fix
+++ a/fs/cifs/cifsglob.h
@@ -479,6 +479,9 @@ struct dir_notify_req {
 #ifdef CONFIG_CIFS_WEAK_PW_HASH
 #define   CIFSSEC_MAY_LANMAN	0x00010
 #define   CIFSSEC_MAY_PLNTXT	0x00020
+#else
+#define   CIFSSEC_MAY_LANMAN	0
+#define   CIFSSEC_MAY_PLNTXT	0
 #endif /* weak passwords */
 #define   CIFSSEC_MAY_SEAL	0x00040 /* not supported yet */
 
_
